app/views/summary.scala.html

```diff
@@ -56,16 +56,16 @@
 				case ersUtil.OPTION_NO => messages("ers.no")
 			})
 		  </td>
-		  <td class="summary-declaration-text-align"><a href="@routes.ReportableEventsController.reportableEventsPage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>
+		  <td class="summary-declaration-text-align"><a aria-label="change-reportable-events" href="@routes.ReportableEventsController.reportableEventsPage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>
 		</tr>
 		@if(reportableEvents == ersUtil.OPTION_YES) {
 			<tr>
 			  <td class="summary-declaration-section-text-align">
 				@Html(if (fileCount == 1) {messages("ers_summary_declaration.file_name")} else {messages("ers_summary_declaration.file_names")})
 			  </td>
 			  <td class="summary-declaration-text-align">@Html(filename)</td>
-			  @if(fileType == ersUtil.OPTION_ODS) {<td class="summary-declaration-text-align"><a href="@routes.FileUploadController.uploadFilePage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>}
-			  @if(fileType == ersUtil.OPTION_CSV) {<td class="summary-declaration-text-align"><a href="@routes.CheckCsvFilesController.checkCsvFilesPage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>}
+			  @if(fileType == ersUtil.OPTION_ODS) {<td class="summary-declaration-text-align"><a aria-label="change-files" href="@routes.FileUploadController.uploadFilePage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>}
+			  @if(fileType == ersUtil.OPTION_CSV) {<td class="summary-declaration-text-align"><a aria-label="change-files" href="@routes.CheckCsvFilesController.checkCsvFilesPage()">@Html(messages("ers_summary_declaration.change_this"))</a></td>}
```

Comment:
do you think it might be better for the user if the type of file they're changing were specified (e.g. 'change-ods-files' and 'change-csv-files')?